### ISSUE TITLE
Incompatible with spatie/laravel-ray

### ISSUE THREAD

**Issue by caugner** [beyondcode/laravel-dump-server#78]
I had installed `beyondcode/laravel-dump-server` in July 2019, and back then it was working fine. Recently I noticed that neither `dump()` nor `dd()` calls showed up anymore in the `artisan dump-server` output.

Today I finally debugged a `dump()` call and noticed that the `VarDumper::$handler` was set to [`spatie/laravel-ray`'s handler](https://github.com/spatie/laravel-ray/blob/c0be902ebe5ea68511a89a5623f6f3ec60abcae8/src/DumpRecorder/DumpRecorder.php#L34-L38) rather than `beyondcode/laravel-dump-server`'s handler:

https://github.com/beyondcode/laravel-dump-server/blob/e27c7b942ab62f6ac7168359393d328ec5215b89/src/DumpServerServiceProvider.php#L52-L54

Now, I actually didn't install `spatie/laravel-ray` myself, but instead it came as an indirect dependency of `psalm/plugin-laravel` (which uses `orchestra/testbench`).

**tl;dr** The `spatie/laravel-ray` package should be marked as incompatible with this package in the `composer.json`, to prevent other users from running into the same trap.

PS: This reminds me to create separate `composer.json` files for tools.
